modules/features/test/test_affine_feature.cpp

```diff
@@ -26,7 +26,7 @@ static bool isSimilarKeypoints( const KeyPoint& p1, const KeyPoint& p2 )
 }
 #endif
 
-TEST(Features2d_AFFINE_FEATURE, regression)
+TEST(Features_AFFINE_FEATURE, regression)
```

Comment:
@asmorkalov The test names in the accuracy tests are renamed too. Should they be reverted as well?

Reply:
Yes.